README.md

```diff
@@ -92,6 +93,36 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion - __experimental__
+
+The ability to convert existing SBOMs means you can create SBOMs in different formats quickly, without the need to regenerate the SBOM from scratch, which may take significantly more time.
+
+``` bash
+syft convert <ORIGINAL-SBOM-FILE> -o <NEW-SBOM-FORMAT>[=<NEW-SBOM-FILE>]
+```
+
+This feature is experimental and data might be lost when converting formats. Packages are the main SBOM component easily transferable across formats, while files and relationships are more likely to lose data. 
+
+The supported formats are:
+- Syft json
+- Syft table
+- SPDX 2.2 json
+- SPDX 2.2 tag-value
+- cycloneDX 1.3 json
+- cycloneDX 1.3 xml
+
+We support formats with wide community usage AND good encode/decode support by Syft. The recent GitHub format is niche and doesn't have a decoder, therefore was excluded.
+
+The exception to the rule above is table format, which is the default output for two reasons:
+1. it matches syft's default SBOM output
+2. it is handy, as a summary, to print SBOMs as tables
+
+Conversion example:
+```bash
+syft alpine:latest -o json=sbom.syft.json
```

Comment:
I'm a little confused and sorry I didn't catch this the first time. Why is convert aliased over the root command and why has the -o flag changed here from supplying a format to this new syntax of `json=sbom.syft.json`?

Reply:
There is no conversion happening here. What I should have explained better is in this example: first generate an SBOM with syft's format, then convert it.

Reply:
Conversion happens right below, line 123

Reply:
@spiffcs to answer your question: the `-o` hasn't changed, today it supports `-o <format>` and `-o <format>=<file>`, with multiple `-o` allowed. this still works with this `convert` command